Mathlib/Order/RelSeries.lean

```diff
@@ -256,7 +256,7 @@ For two types `α, β` and relation on them `r, s`, if `f : α → β` preserves
 def map (p : RelSeries r) (f : r →r s) : RelSeries s where
   length := p.length
   toFun := f.1.comp p
-  step := (f.2 <| p.step .)
+  step := (f.2 <| p.step ·)
```

Comment:
Maybe should be a separate PR? I assume there are more than one

Reply:
rolled back that change, will do a separate PR